saddle-core/src/main/scala/org/saddle/scalar/ScalarTagAny.scala

```diff
@@ -22,6 +22,8 @@ import org.saddle.Buffer
 import org.saddle.na
 
 class ScalarTagAny[T: CLM] extends ScalarTag[T] {
+
+  val classTag = implicitly[CLM[T]]
```

Comment:
@tnielens this can be removed as well, right?

Reply:
Yes guess so.